src/com/projectkorra/projectkorra/firebending/Illumination.java

```diff
@@ -48,7 +65,7 @@ public Illumination(final Player player) {
 			return;
 		}
 
-		if (player.getLocation().getBlock().getLightLevel() < this.lightThreshold) {
+		if (player.getLocation().getBlock().getLightLevel() < this.lightThreshold && (!MODERN || slotsFree(player))) {

```

Comment:
What is this slotsFree method accomplishing?

Reply:
It checks that they have a hand free to use illumination. If they have no free hand to hold a flame, they shouldn't be able to see in the dark

Reply:
Brilliant catch